src/Cpanel/ClientActions.php

```diff
@@ -60,15 +61,21 @@ public function mergeCpanelAndCFDomains()
             $cpanelDomainList = array_merge($cpanelDomainList, $getCpanelDomains['parked_domains']);
         }
 
+        $Punycode = new Punycode();
```

Comment:
If we ran into this issue on wordpress as well is there a possibility of moving this fix into `cloudflare-plugin-backend` so we ensure we handle unicode domains correctly across all plugins?